traffic_control/clients/python/trafficops/trafficops/tosession.py

```diff
@@ -66,7 +66,7 @@ class TOSession(restapi.RestApiSession):
     of loading up dictionaries for request data.
 
     As of now you can see the following URL for API details:
-       https://trafficcontrol.apache.org/docs/latest/development/traffic_ops.html#api for details
+       https://trafficcontrol.incubator.apache.org/docs/latest/development/traffic_ops.html#api for details
```

Comment:
I think the correct URL is: https://traffic-control-cdn.readthedocs.io/en/latest/api/index.html